core/src/main/java/nl/nn/adapterframework/jdbc/dbms/GenericDbmsSupport.java

```diff
@@ -269,102 +266,91 @@ public String getSchema(Connection conn) throws JdbcException {
 		return null;
 	}
 
-	protected boolean doIsTablePresent(Connection conn, String tablesTable, String schemaColumn, String tableNameColumn, String schemaName, String tableName) throws JdbcException {
-		String query="select count(*) from "+tablesTable+" where upper("+tableNameColumn+")=?";
+	public boolean doIsTablePresent(Connection conn, String tablesTable, String schemaColumn, String tableNameColumn, String schemaName, String tableName) throws JdbcException {
```

Comment:
doIsTablePresent() should not be public. It is there to enable subclasses to provide an alternative implementation for isTablePresent, if the generic implementation does not work for them.
Please convert the original implementation using a literal query to a query using parameters.
Please also remove the implementation of of isTablePresent(), (that call doIsTablePresent()) from H2DbmsSupport, MsSqlServerDbmsSupport and OracleDbmsSupport, so that they will use the default using getMetaData()